core-bundle/src/Csp/CspParser.php

```diff
@@ -32,7 +32,7 @@ public function parseHeader(string $header): DirectiveSet
 
         $parser = new ContentSecurityPolicyParser();
         $names = $directiveSet->getNames();
-        $directives = array_filter(array_map(trim(...), explode(';', $header)));
+        $directives = array_filter(array_map(static fn (string $item) => trim(preg_replace('/\s+/', ' ', $item)), explode(';', $header)));
```

Comment:
Wouldn't this also work? At least the tests seem fine.
```suggestion
        $directives = explode(';', preg_replace('/\s+/', ' ', $header));
```

I'm not too familiar with the spec, though, so feel free to ignore. :slightly_smiling_face:

Reply:
You're right, there's a trim anyway later  in the `foreach` 🙃